cmake/modules/FindPETSC.cmake

```diff
@@ -38,11 +38,12 @@ SET_IF_EMPTY(PETSC_DIR "$ENV{PETSC_DIR}")
 SET_IF_EMPTY(PETSC_ARCH "$ENV{PETSC_ARCH}")
 
 DEAL_II_FIND_LIBRARY(PETSC_LIBRARY
-  NAMES petsc
+  NAMES libpetsc
```

Comment:
`NAMES` declares a list of one ore more alternative names. So change this to `NAMES petsc libpetsc` please.